src/main/kotlin/com/team4099/robot2023/commands/drivetrain/FollowPathPlannerPathCommand.kt

```diff
@@ -140,6 +159,9 @@ class FollowPathPlannerPathCommand(val drivetrain: Drivetrain, val path: PathPla
 
   override fun initialize() {
     trajStartTime = Clock.fpgaTime
+    if (shiftByOdoTField) {
+      odoTField = drivetrain.odomTField
+    }
```

Comment:
I don't like this either -- I would prefer if `odomTField` always had the right value, you just used it only when you needed it to. The variable name should mean something, if it says it represents the transform between the `odom` and `field` frames you should make sure it does that, otherwise it will be confusing to debug

Reply:
pretty sure this meant always intiailize it to be `odoTField` and not make it a supplier, made fix.